### ISSUE TITLE
Transform unicode characters in body text

### ISSUE THREAD

**Issue by lizgzil** [wellcometrust/covid19#16]
e.g. "5 \u00b7 2 days" has come out as an answer to "How long is the incubation period for the virus?"

**Comment by nsorros** [comment]
Transform into?

**Comment by lizgzil** [comment]
good point. Actually looking at it I think transformers normalises unicode charcters anyway (https://huggingface.co/transformers/v1.2.0/_modules/pytorch_transformers/tokenization_bert.html (`_run_strip_accents`)).

Maybe its just the output that might not look nice with unicode in then. I think `print` prints unicode characters nicely
